File: java/leetcode/intersection-of-two-arrays.java
class Solution {
    public int[] intersection (int[] nums1, int[] nums2) {
        List<Integer> list = new ArrayList<Integer>();
        for (int i = 0; i < nums1.length; i++) {
            for (int j = 0; j < nums2.length; j++) {
                if (nums1[i] == nums2[j] && !list.contains(nums2[j])) {
                    list.add(nums2[j]);
                }
            }
        }
        int[] newArr = new int[list.size()];
        Iterator<Integer> iterator = list.iterator();
        for (int k = 0; k < newArr.length; k++) {
            newArr[k] = iterator.next().intValue();
        }
        return newArr; // cast to array
    }
}

File: python/leetcode/intersection-of-two-arrays.py
class Solution:
    def intersection(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[int]
        """
        lst = []
        for i in nums1:
            for j in nums2:
                if i == j and j not in lst:
                    lst.append(j)
        return lst
